features/step_definitions/operators.rb

```diff
@@ -0,0 +1,20 @@
+# operators related helper steps
+Given /^all clusteroperators reached version #{QUOTED} successfully$/ do |version|
+  ensure_admin_tagged
+  clusteroperators = BushSlicer::ClusterOperator.list(user: admin)
+  clusteroperators.each do | co |
+    raise "version does not match #{version}" unless co.version_exists?(version: version)
```

Comment:
What if one upgrades only single cluster operator? Should it just check all operators are non-degraded and then separate step to test particular operator is at version `X`? Or how does it work?

Reply:
@akostadinov this method for cluster-wide upgrade check.   I've not tried a single clusteroperator upgarde...not sure if that's possible (need to check).   As the method name indicates it's checking `all clusteropeators`.

Reply:
@akostadinov found out that the only way to update an individual `clusteroperator` is to create a release image with only one operator that was updated... so for non-ART folks, it's not possible to do that.  It's probably a good to check all of them anyway.

Reply:
You sure we never get operators with different versions? Hard to believe for me. Can anybody else comment on such probability?

Reply:
@akostadinov Trevor states that `anyone can create a new release image; you just need a registry somewhere to push it to`.  For our upgrade workflow, we just do a cluster-wide upgrade and not individual clusteroperators.

Reply:
I'm not sure I completely agree here but we can understand better what we need once it is needed so ack.